iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/twin/ReportedPropertiesTests.java

```diff
@@ -144,52 +109,7 @@ public void testUpdateReportedProperties() throws IOException, InterruptedExcept
     @Test
     @StandardTierHubOnlyTest
     @ContinuousIntegrationTest
-    public void testUpdateReportedPropertiesMultiThreaded() throws IOException, InterruptedException, IotHubException
-    {
-        // arrange
-        ExecutorService executor = Executors.newFixedThreadPool(MAX_PROPERTIES_TO_TEST);
-
-        // send max_prop RP all at once
-        testInstance.deviceUnderTest.dCDeviceForTwin.createNewReportedProperties(MAX_PROPERTIES_TO_TEST);
-        testInstance.testIdentity.getClient().sendReportedProperties(testInstance.deviceUnderTest.dCDeviceForTwin.getReportedProp());
-        Thread.sleep(REPORTED_PROPERTIES_PROPAGATION_DELAY_MILLISECONDS);
-
-        // act
-        // Update RP
-        for (int i = 0; i < MAX_PROPERTIES_TO_TEST; i++)
-        {
-            final int index = i;
-            executor.submit(() -> {
-                try
-                {
-                    Set<Property> updatedProperties = testInstance.deviceUnderTest.dCDeviceForTwin.updateExistingReportedProperty(index);
-                    testInstance.testIdentity.getClient().sendReportedProperties(updatedProperties);
-                    waitAndVerifyTwinStatusBecomesSuccess();
-                }
-                catch (IOException | InterruptedException e)
-                {
-                    fail(CorrelationDetailsLoggingAssert.buildExceptionMessage("Unexpected exception occurred during sending reported properties: " + Tools.getStackTraceFromThrowable(e), testInstance.testIdentity.getClient()));
-                }
-            });
-        }
-        Thread.sleep(DELAY_BETWEEN_OPERATIONS);
-        executor.shutdown();
-        if (!executor.awaitTermination(MULTITHREADED_WAIT_TIMEOUT_MILLISECONDS, TimeUnit.MILLISECONDS))
-        {
-            executor.shutdownNow();
-        }
-
-        // assert
-        Assert.assertEquals(CorrelationDetailsLoggingAssert.buildExceptionMessage("Expected OK but twin status was " + testInstance.deviceUnderTest.deviceTwinStatus, testInstance.testIdentity.getClient()), OK, testInstance.deviceUnderTest.deviceTwinStatus);
-
-        // verify if they are received by SC
-        readReportedPropertiesAndVerify(testInstance.deviceUnderTest, PROPERTY_VALUE_UPDATE, MAX_PROPERTIES_TO_TEST);
-    }
-
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void testUpdateReportedPropertiesSequential() throws IOException, InterruptedException, IotHubException
```

Comment:
Github makes it look like I deleted this test, but I just renamed it to "Sequential**ly**"